src/backend/base/langflow/schema/schema.py

```diff
@@ -41,77 +41,63 @@ class OutputValue(BaseModel):
 
 
 def get_type(payload):
-    result = LogType.UNKNOWN
-    match payload:
-        case Message():
-            result = LogType.MESSAGE
-
-        case Data():
-            result = LogType.DATA
-
-        case dict():
-            result = LogType.OBJECT
-
-        case list() | DataFrame():
-            result = LogType.ARRAY
-
-        case str():
-            result = LogType.TEXT
-
-    if result == LogType.UNKNOWN and (
-        (payload and isinstance(payload, Generator))
-        or (isinstance(payload, Message) and isinstance(payload.text, Generator))
+    if isinstance(payload, Message):
```

Comment:
Using a `match` statement seems like a more modern approach here. Is there a performance penalty ?
I would prefer returning from the matched cases and implement a `_` case.